bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml

```diff
@@ -212,14 +168,14 @@
 		<state readOnly="true"></state>
 	</channel-type>
 
-	<channel-type id="powerConsumption" advanced="false">
+	<channel-type id="powerConsumption">
```

Comment:
You suppressed "advanced" to all channel ?
When you have many channels on a thing, this is reasonable to have some of them being advanced.
In general, you don't set a channel as advanced when you are sure that any user will use/link it.

Reply:
No, only for remaining/elapsed time as I think these channels are quite common. almost the primary/most important ones. For the rest I just removed explicit attribute as **advanced=false** is default. This makes it more readable IMHO.

Reply:
Oh you're right, I did not take care that advanced attribute was set to false.
In this case, this is OK for me.